### PR TITLE
[FIX] l10n_ec_website_sale: remove unneeded test inheritance

### PR DESCRIPTION
**Pull request by mattismegevand**
Issues:
The test I created here had an inheritance which was in enterprise, as pointed by gawa https://github.com/odoo/odoo/pull/168930#issuecomment-2340200503

Solution:
Correct the test to remove the inheritance.

opw-3921156

### CODE REVIEW

File: addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
     def test_checkout_address_ec(self):
+        company = self.env['res.company'].create({
+            'name': "EC Test Company",
+            'vat': "1792366836001",
+            'street': "Avenida Machala 42",
+            'zip': "090514",
+            'city': "Guayaquil",
+            'country_id': self.env.ref('base.ec').id,
+        })
+        self.env.company = company
```
Comment (FlorianGilbert) [inline addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py:16]: Could be onelined?

**Review by FlorianGilbert — COMMENTED** [review]
Small remark, otherwise, lgtm

File: addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py (hunk 1 above)
Comment (mattismegevand) [reply addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py:16]: I'd rather not as it would result in a ~= 210 column line.

Comment (mattismegevand) [reply addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py]: I'm stupid ahhah, it's fixed!

File: addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
 @tagged('post_install_l10n', 'post_install', '-at_install')
-class TestUi(HttpCase, TestEcEdiCommon):
+class TestUi(HttpCase, AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
 
     def test_checkout_address_ec(self):
+        self.env.company.country_id = self.env.ref('base.ec').id,
```
Comment (FlorianGilbert) [inline addons/l10n_ec_website_sale/tests/test_l10n_ec_website_sale.py:14]: You'll have a tuple here
```suggestion
        self.env.company.country_id = self.env.ref('base.ec').id
```

**Review by FlorianGilbert — APPROVED** [review]
Thanks, could you please fix the runbot? 

@alialfie Here as well, may i ask you to r+ it for me if it's green?

@robodoo delegate=@alialfie

**Comment by FlorianGilbert** [comment]
@robodoo r+

**Pull request closed without merging**